packages/react-native-editor/__device-tests__/gutenberg-editor-media-blocks-@canary.test.js

```diff
@@ -0,0 +1,165 @@
+		block.click();
+
+		// wait for the media picker's Media Library option to come up
+		await waitForMediaLibrary( editorPage.driver );
+
+		// tap on Media Library option
+		await editorPage.chooseMediaLibrary();
+
+		// get the html version of the content
+		const html = await editorPage.getHtmlContent();
+
+		// verify the html matches the expected content
+		expect( html.toLowerCase() ).toBe(
+			testData.audioBlockPlaceholder.toLowerCase()
+		);
+
+		block = await editorPage.getBlockAtPosition( blockNames.audio );
+		await block.click();
+		await editorPage.removeBlock();
+	} );
+} );
+
+describe( 'Gutenberg Editor File Block tests', () => {
+	it( 'should be able to add a file block and a file to it', async () => {
+		// add a file block
+		await editorPage.addNewBlock( blockNames.file );
+
+		// dismiss the media picker automatically opened when adding a file block
+		await waitForMediaLibrary( editorPage.driver );
+		await editorPage.closePicker();
+
+		// verify there's a file block
+		let block = await editorPage.getFirstBlockVisible();
+		await expect( block ).toBeTruthy();
+
+		// tap on the file block
+		block.click();
+
+		// wait for the media picker's Media Library option to come up
+		await waitForMediaLibrary( editorPage.driver );
+
+		// tap on Media Library option
+		await editorPage.chooseMediaLibrary();
+
+		// get the html version of the content
+		const html = await editorPage.getHtmlContent();
+
+		// verify the html matches the expected content
+		expect( html.toLowerCase() ).toBe(
+			testData.fileBlockPlaceholder.toLowerCase()
+		);
+
+		block = await editorPage.getBlockAtPosition( blockNames.file );
+		await block.click();
+		await editorPage.removeBlock();
+	} );
+} );
+
+// iOS only test - Can only add image from media library on iOS.
+onlyOniOS( 'Gutenberg Editor Image Block tests', () => {
+	it( 'should be able to add an image block', async () => {
+		await editorPage.addNewBlock( blockNames.image );
+		await editorPage.closePicker();
+
+		let imageBlock = await editorPage.getBlockAtPosition(
+			blockNames.image
+		);
+
+		await editorPage.selectEmptyImageBlock( imageBlock );
+		await editorPage.chooseMediaLibrary();
+
+		await editorPage.waitForElementToBeDisplayedById(
+			'A snow-capped mountain top in a cloudy sky with red-leafed trees in the foreground',
+			2000
+		);
+		await editorPage.enterCaptionToSelectedImageBlock(
+			testData.imageCaption,
+			true
+		);
+
+		const html = await editorPage.getHtmlContent();
+
+		expect( html.toLowerCase() ).toBe(
+			testData.imageShortHtml.toLowerCase()
+		);
+
+		imageBlock = await editorPage.getBlockAtPosition( blockNames.image );
+		await imageBlock.click();
+		await editorPage.removeBlock();
+	} );
+} );
+
+onlyOniOS( 'Gutenberg Editor Cover Block test', () => {
+	it( 'should displayed properly and have properly converted height (ios only)', async () => {
+		// Temporarily this test is skipped on Android,due to the inconsistency of the results,
```

Comment:
```suggestion
		// Temporarily this test is skipped on Android, due to the inconsistency of the results,
```

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/48026/commits/bff19d2306ff8244cae576f4f11134029628a52e